LICENSE-3rdparty.csv

```diff
@@ -48,6 +48,7 @@ dev,proxyquire,MIT,Copyright 2013 Thorsten Lorenz
 dev,redis,MIT,Copyright 2016 NodeRedis
 dev,require-dir,MIT,2012-2015 Aseem Kishore
 dev,retry,MIT,Copyright 2011 Tim Koschützki Felix Geisendörfer
+dev,selfsigned,MIT,Paolo Fragomeni and Charles Bushong
```

Comment:
Added a dev dependency here, hope that’s ok

Reply:
It's ok but it's missing `Copyright <years>` in front of the names.

Reply:
I can add the `Copyright` part because that’s implicit, but they have not put any years on that themselves. (Or at least I couldn’t find a copyright line.)

I’ll ask them in an issue.

Reply:
https://github.com/jfromaniello/selfsigned/issues/28